modules/kn-service-describe.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-describe_{context}"]
+= Describing serverless applications
+
+You can describe a Knative service by using the `kn` CLI `kn service describe` command.
+
+.Example commands
+
+* Describe a service:
++
+[source,terminal]
+----
+$ kn service describe <service-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         example-service
+Namespace:    knative-serving
```

Comment:
I don't quite like the example here for several reasons. 
1. the namespace is knative-serving, which is not usually a namespace the user would put their own services into, so it is kind of misleading
2. the annotations are again very specific and not examply (they match the kn-cli service, but that is not a service that the user would create)
3. the image looks like something that exists, but isn't.

IMHO we should just use an example created by https://docs.openshift.com/container-platform/4.7/serverless/serving-creating-managing-apps.html.

```
Name:       hello
Namespace:  default
Age:        3m
URL:        http://hello-default.apps.ocp.example.com

Revisions:  
  100%  @latest (hello-00001) [1] (3m)
        Image:  docker.io/openshift/hello-openshift (at aaea76)

Conditions:  
  OK TYPE                   AGE REASON
  ++ Ready                   3m 
  ++ ConfigurationsReady     3m 
  ++ RoutesReady             3m 
```

Reply:
👍 , in https://github.com/openshift/openshift-docs/pull/31518#discussion_r617694972 I suggested to followup with another PR that not only fixes the output issues here, but also adds a sample to the verbose output.

But let me try to find a slot today to provide the sample output right now, just spinning up a cluster ....

Reply:
@abrennan89 Hier is the ouput for `kn service describe` with and without `--verbose`, with the example the @maschmid mentioned above:

```
Name:       hello
Namespace:  default
Age:        2m
URL:        http://hello-default.apps.ocp.example.com

Revisions:
  100%  @latest (hello-00001) [1] (2m)
        Image:  docker.io/openshift/hello-openshift (pinned to aaea76)

Conditions:
  OK TYPE                   AGE REASON
  ++ Ready                   1m
  ++ ConfigurationsReady     1m
  ++ RoutesReady             1m
```

with `--verbose`:

```
Name:         hello
Namespace:    default
Annotations:  serving.knative.dev/creator=system:admin
              serving.knative.dev/lastModifier=system:admin
Age:          3m
URL:          http://hello-default.apps.ocp.example.com
Cluster:      http://hello.default.svc.cluster.local

Revisions:
  100%  @latest (hello-00001) [1] (3m)
        Image:  docker.io/openshift/hello-openshift (pinned to aaea76)
        Env:    RESPONSE=Hello Serverless!

Conditions:
  OK TYPE                   AGE REASON
  ++ Ready                   3m
  ++ ConfigurationsReady     3m
  ++ RoutesReady             3m
```